fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -519,7 +516,7 @@ class HomeFragment : Fragment() {
             homeActivity = activity as HomeActivity,
             navController = findNavController(),
             menuButton = WeakReference(binding.menuButton),
-            hideOnboardingIfNeeded = ::hideOnboardingIfNeeded,
+            hideOnboardingIfNeeded = {},
```

Comment:
assuming this unnecessary lambda will be cleaned up?

Reply:
Not necessarily. I have added an optional parameter to clean this up a bit. We still need this function for the onboarding fragment which will need to call `onboarding.finish()` and navigate to the home fragment to add it to the back stack. This is for when the user interacts with the home menu from the onboarding, and navigates to one of the fragments depending on the menu item selection.